common/src/subgraph_client.rs

```diff
@@ -20,48 +19,32 @@ pub struct SubgraphClient {
 }
 
 impl SubgraphClient {
-    pub fn new(
-        graph_node_query_endpoint: Option<&str>,
-        deployment: Option<&DeploymentId>,
-        subgraph_url: &str,
-    ) -> Result<Self, anyhow::Error> {
-        // TODO: Check indexing status of the local subgraph deployment
-        // if the deployment is healthy and synced, use local_subgraoh_endpoint
-        let _local_subgraph_endpoint = match (graph_node_query_endpoint, deployment) {
-            (Some(endpoint), Some(id)) => Some(Self::local_deployment_endpoint(endpoint, id)?),
-            _ => None,
-        };
-
-        let subgraph_url = Url::parse(subgraph_url)
-            .map_err(|e| anyhow!("Could not parse subgraph url `{}`: {}", subgraph_url, e))?;
+    pub fn new(name: &str, query_url: &str) -> Result<Self, anyhow::Error> {
```

Comment:
Do we really need `name`? It seems to be only used for error messages.

Reply:
Yeah, no, let's not include it.